<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_CreerProfil</class>
 <widget class="QDialog" name="f_CreerProfil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>345</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Lb_MdP">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>90</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Mdp">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>68</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_NomProfil">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>18</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_NomProfil">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>90</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom du profil :</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="BtBx_Valider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="Fr_Config">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>260</width>
     <height>15</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="RBt_OConfig">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Oui</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RBt_NConfig">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Non</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Config">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Configuration :</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Fr_Gestion">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>260</width>
     <height>15</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="RBt_OProfil">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Oui</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RBt_NProfil">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Non</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Gestion">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestion de profill :</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Fr_ProfilDefaut">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>260</width>
     <height>15</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Lb_ProfilDefaut">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>90</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Profil par d√©faut :</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RBt_ODefaut">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Oui</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RBt_NDefaut">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Non</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="CkBx_NormeOrga">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Affichage nouvelle norme d'orgagniramme</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
